solidity-v1/dashboard/src/services/threshold-authorization.service.js

```diff
@@ -0,0 +1,101 @@
+import {
+  ContractsLoaded,
+  getContractDeploymentBlockNumber,
+  getThresholdTokenStakingAddress,
+} from "../contracts"
+import { getAllOperatorStakedEventsByAuthorizer } from "./token-staking.service"
+import {
+  AUTH_CONTRACTS_LABEL,
+  TOKEN_GRANT_CONTRACT_NAME,
+} from "../constants/constants"
+import { Keep } from "../contracts"
+import { isSameEthAddress } from "../utils/general.utils"
+import { gt } from "../utils/arithmetics.utils"
+
+const fetchThresholdAuthorizationData = async (address) => {
+  if (!address) {
+    return []
+  }
+  const thresholdTokenStakingContractAddress = getThresholdTokenStakingAddress()
```

Comment:
I think we can use `Keep.thresholdStakingContract.address` and get rid of the `getThresholdTokenStakingAddress `.